typo on bulletproofs.md (#1523)
monero-project/monero-site: _i18n/en/resources/moneropedia/bulletproofs.md

The code has been written and rewritten to follow the new version of bulletproofs which was still being developed, but once this Monero implementation was finalized, the resulting deployment should be taken with extreme care.  
Therefore, the community started an auditing process. Researchers reached out to Benedikt Bünz, lead author of the Bulletproofs paper, and to [OSTIF](https://ostif.org/) an organization which helps open source technologies to improve and secure themselves.  
OSTIF directed the group to several organizations with the skills required to perform the audit. While one of them asked to be kept unnamed and was therefore put away from the process that needed to be public, two others (QuarksLab & Kudelski Security) were choosen to conduct the audit.  

> Our 3 auditors were funded by the community to check out if the implementation did not contain critical bugs, and if it did not have any exploits.  

Reviewer: While you're at it, I would also suggest improving the readability of the sentence:
```suggestion
Our 3 auditors were funded by the community to ensure that the implementation did not contain any critical bugs or exploits.
```
